src/babylon.scene.ts

```diff
@@ -1983,6 +1983,8 @@
             this._boundingBoxRenderer.reset();
             this._edgesRenderers.reset();
 
+            this.activeCamera.computeFrustumPlanes(this._frustumPlanes);

```

Comment:
I'm a little bit concern by this. I do not like having a call just for a specific camera. (and it is weird as other cameras also have frustum planes)